R/plotSpectra-PTM.R

```diff
@@ -151,34 +150,34 @@ plotSpectraPTM <- function(x, deltaMz = TRUE, ppm = 20,
     nsp <- length(x)
     old_par <- par(no.readonly = TRUE)
     on.exit(par(old_par))
-
+    
     if (length(main) != nsp) main <- rep(main[1], nsp)
-
+    
     labels <- labelFragments(x, ppm = ppm, what = "ion", ...)
-
-    if (deltaMz) { ## Generate deltaMzData labels for .plot_single_spectrum_PTM
+    
+    if (deltaMz) { ## Generate deltaMzData labels for .plot_single_spectrum_PTM_dev
         deltaMzData <- labelFragments(x, ppm = ppm, what = "mz", ...)
         layout_matrix <- .make_layout_matrix(length(labels))
         layout(layout_matrix,
                heights = rep(c(5, 1), length.out = nrow(layout_matrix)))
     } else {
-      par(mfrow = n2mfrow(length(labels), asp = asp))
-      deltaMzData <- NULL
+        par(mfrow = n2mfrow(length(labels), asp = asp))
+        deltaMzData <- NULL
     }
     spectrum_number <- attr(labels, "group")
-
+    
     for (i in seq_along(spectrum_number)) {
-        .plot_single_spectrum_PTM(x[spectrum_number[i]], xlab = xlab,
-                                  ylab = ylab, xlim = xlim,
-                                  ylim = ylim,
-                                  main = main[spectrum_number[i]],
-                                  col = col, labels = labels[i],
-                                  labelCex = labelCex, labelSrt = labelSrt,
-                                  labelAdj = labelAdj, labelPos = labelPos,
-                                  labelOffset = labelOffset,
-                                  minorTicks = minorTicks,
-                                  deltaMzData = deltaMzData[[i]],
-                                  ppm = ppm, old_par = old_par)
+        .plot_single_spectrum_PTM_dev(x[spectrum_number[i]], xlab = xlab,
```

Comment:
why is there a `_dev` in the function name (see also comment line 157). This sounds like you named it like that during development, but it should get a proper name before merging.